src/preflight/internal-links.js

```diff
@@ -48,7 +51,12 @@ export async function runInternalLinkChecks(scrapedObjects, context, options = {
           try {
             const res = await fetch(href, {
               method: 'HEAD',
-              ...options,
+              redirect: 'manual',
```

Comment:
@dzehnder , Just curious - why is this needed? It effectively prevents following any redirects.